### ISSUE TITLE
Сохранение тредов. 

### ISSUE THREAD

**Issue by dgorodnichybak** [vortexwolf/2ch-Browser#11]
Если будет синхронизация с дропбоксом то вообще огонь.

**Comment by kasthack** [comment]
Таки да, кода не особо много, в  кэше ведь полноценное сохранение было.

**Comment by vortexwolf** [comment]
На самом деле кода будет много, т.к. под сохранением тредов все понимают именно сохранение в *.html с последующей возможностью открыть в обычном браузере.

**Comment by kasthack** [comment]
Я был бы рад просто возможности открыть сохраненный тред в 2ch-браузере.
В крайнем случае могу конвертер на шарпике запилить

**Comment by dgorodnichybak** [comment]
Мне собтвенно говоря тоже html не нужен. И точно так же могу сделать конвертер на Ruby. Хочется читать вины оффлайн.

**Comment by nplusp** [comment]
И в самом деле. Классно было бы сохранить весь тред.

**Comment by vortexwolf** [comment]
Я так и не придумал, куда сохранять thumbnails и нужно ли сохранять изображения в полном размере. Да и в целом не все так просто, т.к. сохранение в оффлайне отличается от кеширования.

**Comment by dgorodnichybak** [comment]
Мне лично изображения в полном размере не нужны. Главное чтобы можно было бы файлы сохраненных тредов перенести на другой андроид и оно заработало.

**Comment by magicgoose** [comment]
@vortexwolf  Не совсем понял, чем сохранение в оффлайне отличается от кеширования? Можно просто кеш взять и всё, что касается данного треда в какой-то контейнер пихнуть, хоть в обычный zip.

**Comment by vortexwolf** [comment]
@magicgoose Нужно сохранить все thumbnails, добавить страницу со списком сохраненных тредов, и отдельную страницу для просмотра, которая отличается от страницы просмотра обычного треда.

**Comment by vortexwolf** [comment]
Между прочим, есть приложение [Pocket](https://play.google.com/store/apps/details?id=com.ideashower.readitlater.pro). В любом треде открываешь меню "Поделиться" и выбираешь Pocket из списка.

**Comment by nplusp** [comment]
@vortexwolf да, есть. Но треды хранятся ровно до того момента, пока есть на сайте, потом умирают.
